src/GenerateProgrammaticPipelineFromConfig/Generator.php

```diff
@@ -240,10 +239,11 @@ private function generatePipeline(array $config)
             $error      = isset($spec['error']) ? (bool) $spec['error'] : false;
 
             if ($error) {
-                $this->console->writeLine(sprintf(
+                $console = $this->getErrorConsole($this->console);
```

Comment:
Would suggest just printing to `$this->console`, unless STDOUT is important in this generator